crypto/bls12381_utils_test.go

```diff
@@ -51,7 +51,7 @@ func TestPRGseeding(t *testing.T) {
 func BenchmarkScalarMultG1G2(b *testing.B) {
 	blsInstance.reInit()
 	seed := make([]byte, securityBits/8)
-	_, _ = rand.Read(seed)
+	_, _ = crand.Read(seed)
```

Comment:
can we check that error is nil?

Reply:
In these bench functions, if randomness fail, we would measure the timing on the same inputs instead of random inputs. 
I'll add the check to avoid confusion.